services/lsst-login.rst

```diff
@@ -89,9 +89,93 @@ When using an ``lsst-login`` node as a "jump host" you may also wish to configur
       DynamicForward yourportnumber
 
-You may also wish to reuse a single connection to/through an ``lsst-login`` node via a control socket/multiplexing. See for example
+You may also wish to reuse a single connection to/through an ``lsst-login`` node via a control socket/multiplexing. This means you can authenticate to the login node once, and reuse that connection in a 2nd terminal without authenticating again. See for example
 `OpenSSH Cookbook - Multiplexing <https://en.wikibooks.org/wiki/OpenSSH/Cookbook/Multiplexing>`_.
 
+A relatively complete ``~/.ssh/config`` "recipe" for streamlining your SSH connections (assuming OpenSSH, e.g., on Linux or macOS) through the ``lsst-login`` nodes might look like this:
+
+.. prompt:: bash $ auto
+
+   # Set common config for the lsst-login nodes
+   Host lsst-login*
+      # if your account on your local workstation/laptop does not match your LSST username, indicate the latter should be used;
+      # substitute your own NCSA username
+      User ncsausername               
+      # allow use of a Kerberos ticket on your local machine for auth to LSST machines
+      GSSAPIAuthentication yes   
+      # prefer Kerberos ticket auth, amongst other possibilities (order/include others as desired)
+      PreferredAuthentications gssapi-with-mic,keyboard-interactive,password
+      # forward your local Kerberos ticket to the login node if you need to continue to another LSST server after the login
+      GSSAPIDelegateCredentials yes
+      # configure OpenSSH Control Master "multiplexing" (to allow reuse of an initial connection)
+      ControlMaster auto
+      ControlPath ~/.ssh/cm_socket/%r@%h:%p
+      ControlPersist 24h
+
+   # Define aliases onto full hostnames for each login node
+   Host lsst-login01
+      HostName lsst-login01.ncsa.illinois.edu
+   Host lsst-login02
+      HostName lsst-login02.ncsa.illinois.edu
+   Host lsst-login03
+      HostName lsst-login03.ncsa.illinois.edu
+
+   # Define an alias and config for an internal node, which can only be reached through a login node
+   Host lsst-cnd-sub01
+      HostName lsst-condordac-sub01.ncsa.illinois.edu
+      # you may need to specify your NCSA username again
+      User ncsausername
+      # when connecting to this internal host, tunnel/jump through a login node (using an alias you defined above)
+      ProxyJump lsst-login01
+      # if you want to use your local Kerberos ticket to authenticate on the interior node, configure that:
+      GSSAPIAuthentication yes
+      PreferredAuthentications gssapi-with-mic
+      # if the internal node is a batch submit node where you might want a Kerberos ticket (e.g., to
+      # submit jobs to HTCondor), you can choose to forward your credentials:
+      GSSAPIDelegateCredentials yes
+      # if you need to configure port forwarding to the internal node, you can do that here;
+      # substitute your actual port number
+      DynamicForward yourportnumber
```

Comment:
This DynamicForward isn't working for me. In other examples I have `DynamicForward localhost: yourportnumber`, but that isn't working for me here either.

Reply:
Nevermind, this part is working. I actually had an issue elsewhere with the `ControlPath` setup.